source/renderer/app/themes/overrides/LinkOverrides.scss

```diff
@@ -1,3 +1,3 @@
 .root {
-  // display: inline-block;
+  font-size: 14px;
```

Comment:
Could/should we control this using CSS variables?

Reply:
Yes we could also the `--rp-link-label-size` but it's more work/changes to add this to all theme files + it's consistent for all themes … I'm not 100% sure yet but i think maybe we should only use CSS variables for parts that can be different per theme?

Ofc this topic is a bit deeper than it looks here … because overrides like this are supported by react-polymorph but less obvious if they get more complicated. Css vars are like a "official" theme config but require more copy/paste and work from our side.

Reply:
Got it, I think at this point it might be just too complex for what we're trying to accomplish, perhaps let's stick to the current implementation